File: Serene3/Serene3.Web/Modules/ServerTypes/com/orderForm.ts
import { BooleanEditor, DateEditor, initFormType, IntegerEditor, PrefixedContext, ServiceLookupEditor, StringEditor } from "@serenity-is/corelib";

export interface orderForm {
    Userid: StringEditor;
    Name: StringEditor;
    Phone: StringEditor;
    Email: StringEditor;
    Onlinepaid: BooleanEditor;
    EntityDate: StringEditor;
    CustomerName: StringEditor;
    CountryId: ServiceLookupEditor;
    StateId: ServiceLookupEditor;
    CityId: ServiceLookupEditor;
    Address: StringEditor;
    OrderDate: DateEditor;
    Productid: IntegerEditor;
}

export class orderForm extends PrefixedContext {
    static readonly formKey = 'com.order';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!orderForm.init) {
            orderForm.init = true;

            var w0 = StringEditor;
            var w1 = BooleanEditor;
            var w2 = ServiceLookupEditor;
            var w3 = DateEditor;
            var w4 = IntegerEditor;

            initFormType(orderForm, [
                'Userid', w0,
                'Name', w0,
                'Phone', w0,
                'Email', w0,
                'Onlinepaid', w1,
                'EntityDate', w0,
                'CustomerName', w0,
                'CountryId', w2,
                'StateId', w2,
                'CityId', w2,
                'Address', w0,
                'OrderDate', w3,
                'Productid', w4
            ]);
        }
    }
}

File: Serene3/Serene3.Web/Modules/com/order/orderForm.cs
using Serenity.ComponentModel;
using System;

namespace Serene3.com.Forms;

[FormScript("com.order")]
[BasedOnRow(typeof(orderRow), CheckNames = true)]
public class orderForm
{
    public string Userid { get; set; }
    public string Name { get; set; }
    public string Phone { get; set; }
    public string Email { get; set; }

    public bool Onlinepaid { get; set; }
    public DateOnly EntityDate { get; set; }
    public string CustomerName { get; set; }
    public int CountryId { get; set; }
    public int StateId { get; set; }
    public int CityId { get; set; }
    public string Address { get; set; }
    public DateTime OrderDate { get; set; }
    public int Productid { get; set; }
}